### ISSUE TITLE
Jammin' with GDevelop

### ISSUE THREAD

**Issue by davidwesst** [davidwesst/blog#30]
---

title: "Jammin' with GDevelop (in GitHub Game Off 2021)"

---

I've been feeling the urge to make some games, so I'm going back to where my game development story started and entering the GitHub Game Off this November. 

In doing so, I've decided to take focus on game design and less on code. To do that I'm going to be using GDevelop to create the game, and try and challenge myself by attempting _not to focus on code or devops_ for the duration of the gamejam.

## Challenge Accepted! No Code or DevOps

I can code and I can DevOps. In fact, one might say these are my strengths. This is why I want to avoid them-- I want to focus on my weaker areas with audio, art, and game design.

This is why I decided to go with GDevelop for the jam instead of Unity of Godot. GDevelop seems to be focused on making games work and avoids code up front, rather that the others which are more code-first environments.

If I was able to create graphics, music, and gameplay with Godot over a weekend with [Out the Door](https://davidwesst.itch.io/out-the-door), I'm guessing I should be able to do it over a month.

## Other Tools

I'm going to use as much GDevelop functionality as possible, but just in case I need something else...

### [Pixel] Art 
I have a [Aseprite]( https://www.aseprite.org), [Piskel](https://www.piskelapp.com), or [Pixelorama](https://www.orama-interactive.com/artstation) if I'm getting adventurous.

### Sound and Music
For audio, I'm going to stick with what I know.

For music, I have my partner's electric piano to meddle with a melody and chords, and [Bosca Ceoil](https://boscaceoil.net)

For sound effects, I'm going to stick wtih [Bfxr](https://iznaut.itch.io/bfxr) because it works, and works well for my purposes.

### Planning
I know this is the _**GitHub**_ Game Off and all, but I'm going to avoid using [GitHub issues](https://docs.github.com/en/issues) and use [Codecks](https://www.codecks.io).

I even have access to the new [GitHub Projects (beta)](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects) features, but this is _exactly_ why I want to spend the time on the jam focusing on the game and not the development toys that I love to play with.

## TL;DR; | Conclusion
I'm going back to the month-long GitHub Game Off gamejam to try and make a game with GDevelop. My theory is that forcing myself to not get overly focused on applying my code and DevOps skills, should hopefully help either strengthen or, at the very least let me test my other creative skills.

I'm excited to try and design another game experience. I've already got a few things in my mind's eye, but we'll have to wait and see for when the theme is announced on November 1. 

Thanks for playing.

~ DW

**Comment by davidwesst** [comment]
Won't post this considering I dropped out of the game jam.

**Comment by davidwesst** [comment]
Might be useful reference for #26 and how you should do something first.
